libraries/Microsoft.Bot.Connector/Teams/TeamsOperations.cs

```diff
@@ -719,33 +620,27 @@ private static RetryParams HandleThrottlingException(Exception ex, int currentRe
             }
         }
 
-        private async Task<HttpOperationResponse<string>> SendMessageToAllUsersInTenantWithRetryAsync(IActivity activity, string tenantId, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        private async Task<HttpOperationResponse<T>> GetBatchResponseWithRetryAsync<T>(string apiUrl, string httpMethod, string invocationId, object content = null, Dictionary<string, List<string>> customHeaders = null, string continuationToken = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
As this method is now used for all the operations it should be called:
```suggestion
        private async Task<HttpOperationResponse<T>> GetResponseAsync<T>(string apiUrl, string httpMethod, string invocationId, object content = null, Dictionary<string, List<string>> customHeaders = null, string continuationToken = null, CancellationToken cancellationToken = default(CancellationToken))
```